### DISCUSSION TITLE
Column delete - title length error [Jul 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2761] [Discord Threads]
> 🙋‍♂️ **Natali Nachteule**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@938150704588152832> Thank you for logging the details!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/996329850983948328)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
